Documentation/Setup/Config/Index.rst

```diff
@@ -1625,6 +1636,10 @@ locale\_all
          :ref:`data-type-string`
 
    Description
+        This setting will be removed in v10. In v9, language settings
+        from the :ref:`Site configuration <t3coreapi:sitehandling>` are used by default and the TypoScript settings
```

Comment:
It might make more sense to link to this page: https://docs.typo3.org/m/typo3/reference-coreapi/9.5/en-us/ApiOverview/SiteHandling/AddLanguages.html, rather than to the general Site handling page.

Reply:
I think we should use the `deprecated` directive here